appveyor.yml

```diff
@@ -18,7 +18,7 @@ install:
   # by default, all script lines are interpreted as batch
 
 build:
-  project: g3log\build\g3log.sln
+  project: c:\projects\g3log\build\g3log.sln
```

Comment:
In fact the first option did not work for me. I put the absolute path to make it independent of any change of working directory in the script.